netbox_routing/models/static.py

```diff
@@ -66,9 +66,7 @@ class Meta:
         )
 
     def __str__(self):
-        if self.vrf is None:
-            return f'{self.prefix} NH {self.next_hop}'
-        return f'{self.prefix} VRF {self.vrf} NH {self.next_hop}'
+        return self.name
```

Comment:
```suggestion
            return f'{self.prefix} NH {self.next_hop}'
        return f'{self.prefix} VRF {self.vrf} NH {self.next_hop}'
```

We don't need to alert this